signature/policy_eval_signedby.go

```diff
@@ -47,20 +47,22 @@ func (pr *prSignedBy) isSignatureAuthorAccepted(image types.UnparsedImage, sig [
 		return sarRejected, nil, err
 	}
 	defer os.RemoveAll(dir)
-	mech, err := newGPGSigningMechanismInDirectory(dir)
-	if err != nil {
-		return sarRejected, nil, err
-	}
+	/*
+		mech, err := newGPGtypes.SigningMechanismInDirectory(dir)
+		if err != nil {
+			return sarRejected, nil, err
+		}
+	*/
```

Comment:
@mtrmac the build tags are the "last resort" to me as the management of them proved cumbersome in openshift (we forgot to add it for registry that resulted in storage driver disabled in release, etc..) They also need to be documented as they are exposed to downstream (origin).

The cleanest (in go way) solution would be to separate the packages and have interface for them. The interface (SigningMechanism) is not really a good interface if an implementation is used instead anyway of it.